R/pkg/R/mllib.R

```diff
@@ -41,7 +41,8 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df)
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family=c("gaussian","binomial"))
```

Comment:
Let me know, if only "gaussian" will be sufficient for this example or if I should change the example so that it works for both, thanks!